packages/medusa/src/services/order.ts

```diff
@@ -43,6 +44,9 @@ import RegionService from "./region"
 import ShippingOptionService from "./shipping-option"
 import ShippingProfileService from "./shipping-profile"
 import TotalsService from "./totals"
+import { NewTotalsService, TaxProviderService } from "./index"
+
+export const ORDER_CART_ALREADY_EXISTS_ERROR = "Order from cart already exists"
```

Comment:
thought(non-blocking): Should we change this to an error type or code to not rely on a correct error message in the strategy? 

If so, I think we should move it to `medusa-core-utils/src/errors.ts`